apps/wfo-ui-surf/components/WfoServiceTicketDetailPage/utils.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { EuiIcon } from '@elastic/eui';
+
+export enum ServiceTicketTabIds {
+    GENERAL_TAB = 'general-id',
+    NOTIFICATION_LOG = 'notification-log-id',
+    SENT_EMAILS = 'sent-emails-id',
+}
+
+export const tabs = [
```

Comment:
I would like to suggest to not use something like a utils.tsx. Preferably only .ts files. That means you will have to move this object to the component where it is actually used.